packages/sodium/lib/src/api/sodium_exception.dart

```diff
@@ -21,6 +21,16 @@ class SodiumException implements Exception {
     }
   }
 
+  @internal
+  static void checkSucceededInitInt(
```

Comment:
[optional] Please add a unit-test

Reply:
Added a unit test (ensures both 0 and 1 are successful).